packages/accordion/src/Accordion.tsx

```diff
@@ -0,0 +1,48 @@
+import { OpenIcon, SubtractIcon } from '@littlespoon/ui/icons'
```

Comment:
You should import from the package instead of ui:

```suggestion
import OpenIcon from '@littlespoon/icons/lib/OpenIcon'
import SubtractIcon from '@littlespoon/icons/lib/SubtractIcon'
```